docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -101,6 +101,44 @@ Maximum number of concurrent calls to the rendering service.
 
 Scale factor for rendering images. Value `2` is enough for monitor resolutions, `4` would be better for printed material. Setting a higher value affects performance and memory.
 
+## [auditing]
```

Comment:
Please add a brief intro and a link to the auditing topic.

Reply:
There seems to be no intro in the configuration file documentation, I can't see any in this file or in the global Grafana configuration file. Do you want to change this?

Reply:
It would be a nice change, since it would help the user orient as they go along. 

Per the Grafana documentation style guide, we generally try to avoid stacked headings.